framework/src/main/java/org/apache/drill/test/framework/TestCaseModeler.java

```diff
@@ -86,6 +86,8 @@ public static TestCaseModeler createTestCase(String testId, String type,
     public String outputFormat;
     @JsonProperty("expected-file")
     public String expectedFile;
+    @JsonProperty("fail-extension")
+    public String failExtension;//Define the fail extensions like .fail / .failing in the test definition file 
```

Comment:
Update comment to give space after statement, and as follows: "Failed tests are expected to be tagged with .fail extension. This can be overridden in the test definition file. However, it's not recommended."